apps/prairielearn/src/lib/github.ts

```diff
@@ -212,14 +168,53 @@ export async function createCourseRepoJob(
     job.info(`Creating course ${options.short_name}`);
     job.info(JSON.stringify(options, null, 4));
 
-    // Create base github repo from template
-    job.info('Creating repository from template');
-    await createRepoFromTemplate(client, options.repo_short_name, config.githubCourseTemplate);
+    // Create an empty repository for the course
+    job.info('Creating empty repository');
+
+    await createEmptyRepository(client, options.repo_short_name);
     job.info(`Created repository ${options.repo_short_name}`);
 
+    job.info('Creating infoCourse.json');
+
+    const infoCoursePath = path.join(TEMPLATE_COURSE_PATH, 'infoCourse.json');
+
+    const infoCourse = JSON.parse(await fs.readFile(infoCoursePath, 'utf-8'));
+
+    infoCourse.uuid = uuidv4();
+    infoCourse.name = options.short_name;
+    infoCourse.title = options.title;
+    infoCourse.timezone = options.display_timezone;
+
+    const newContents = JSON.stringify(infoCourse, null, 4);
```

Comment:
We should be using Prettier to format these files like we did in the creation modal PRs.

Reply:
No need; we only want to use Prettier to format JSON files that we're actually writing to disk, mostly so that the "Reformat" button on the file editor page (and any automated edits that use Prettier formatting) won't create unnecessary diffs, as the file will have started out formatted correctly:

https://github.com/PrairieLearn/PrairieLearn/blob/a9e1662d7b12f1b94a5049330fef6a3832490c28/apps/prairielearn/assets/scripts/instructorFileEditorClient.ts#L91-L97